README.md

```diff
@@ -1,33 +1,14 @@
 Active Directory Authentication Library (ADAL) for JavaScript
 ====================================
-[![Build Status](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-js.svg?branch=master)](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-js)
+|[Getting Started](https://github.com/Azure-Samples/active-directory-javascript-singlepageapp-dotnet-webapi)| [Docs](https://aka.ms/aaddev)| [Samples](https://github.com/azure-samples?query=active-directory)| [Support](README.md#community-help-and-support)
+| --- | --- | --- | --- |
 
 Active Directory Authentication Library for JavaScript (ADAL JS) helps you to use Azure AD for handling authentication in your single page applications.
 This library is optimized for working together with AngularJS.
```

Comment:
this library works both with plain JavaScript and Angular JS ?

Reply:
Good catch.